### PR TITLE
docs(README): add release section, screenshot, short installing section

### PR DESCRIPTION
**Pull request by PaulHax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,55 +38,36 @@ git clone https://github.com/vicentebolea/nrtk_explorer_datasets.git
...
 
 [1]: https://trame.readthedocs.io/en/latest/
 [2]: https://www.kitware.com/
 [3]: https://cocodataset.org/
+
+### Create release
+
+1. Merge `main` to `release` with a _merge commit_.
+2. Run "Create Release" workflow with workflow from `release` branch.
+3. Merge `release` back to `main`
```
Comment (PaulHax) [inline README.md:73]: @vicentebolea If we don't do this step right now, we don't get the latest release tag on main until we make another pr merge on release, right?

Comment (vicentebolea) [reply README.md:73]: We have a CI job that does the step 3 automatically

Comment (PaulHax) [reply README.md:73]: Wondering if the semantic release new version commit push will trigger this: https://github.com/Kitware/nrtk-explorer/blob/main/.github/workflows/sync_branches.yaml#L4-L6

Comment (vicentebolea) [reply README.md:73]: Yes, the workflow is: semantic release makes a new commit, this new commit is pushed to the release branch, then the ci job for syncing the branch gets triggered.

**Comment by jourdain** [comment]
I think we still need an install section, maybe smaller/compact?

File: README.md [hunk 2]
```diff
@@ -61,36 +34,49 @@ curl -OL https://github.com/Kitware/nrtk-explorer/archive/refs/heads/main.zip
 
 Inside the application source code top directory, install it with:
 
-```
+```bash
 pip install -e .
```
Comment (jourdain) [inline README.md:38]: Use package name instead. The `-e .` is more for dev setup which is covered below.

Comment (PaulHax) [reply README.md]: I think the `-e .` line was intended as a second step to  `curl` style source code "checkout."  

I just delete those 2 steps.
